ddp-workspace/projects/ddp-pancan/src/assets/i18n/en.json

```diff
@@ -201,92 +208,104 @@
+                    ]
+                },
+                {
+                    "Question": "What does Count Me In do with the data?",
+                    "Paragraphs": [
+                        "After removing all personal information that would readily identify individuals, data is shared freely in an effort to accelerate discoveries, enabling researchers to identify patterns in data. Batches of data are released publicly on a regular basis, and are available to all researchers—so that everyone may use the data to make discoveries and advance progress against cancer, including new medicines and diagnostics.",
+                        "Quick links to released data: \n<a class='Link' href='https://www.mbcproject.org/data-release'>https://www.mbcproject.org/data-release</a>\n<a class='Link' href='https://ascproject.org/data-release'>https://ascproject.org/data-release</a>"
+                    ]
+                },
+                {
+                    "Question": "How have patients been involved in Count Me In?",
+                    "Paragraphs": [
+                        "Count Me In has worked closely with patients, parents, loved ones and patient advocates within several cancer communities from the beginning of the first project, the Metastatic Breast Cancer Project, launched in 2015. Patients have been included in all aspects of the design including concept, goals, messaging, forms, and the website."
+                    ]
+                },
+                {
+                    "Question": "How is Count Me In addressing inclusivity in participation?",
+                    "Paragraphs": [
+                        "Count Me In understands that many barriers exist - structural and historical - that prevent underrepresented and marginalized communities from directly participating in research. Count Me In is committed to the core principle that every cancer patient, regardless of where they live, what language they speak, what their heritage may be - that their experience matters and the inclusion of their experience is imperative in furthering our understanding of cancer.",
+                        "Traditional cancer research does not lend itself to being as inclusive as it should be. Cancer disparities exist as evidence of how much more work in this space needs to be done - and it’s one in which we recognize that we’ll never truly arrive - but continue to strive for, alongside patients. We believe patients should be direct partners in cancer research and know that we can only advance our understanding of cancer and accelerate discoveries, if we do so together."
+                    ]
+                },
+                {
+                    "Question": "How can I receive updates on Count Me In?",
+                    "Paragraphs": [
+                        "Please <a class='Link' href='/join-list'>sign up for our mailing list</a> to receive updates"
```

Comment:
well, I thought about this refactoring and discussed it with Marco, and we decided that it's better to keep internal links outside of json file since they force the page to reload, unlike the routerlink. On the other hand, we have the directive which fixes this behavior for the links which we get from the backend and it should be applied to the translation content automatically, plus you already finished this refactoring, I'd probably keep it. @MocanaAtBroad what do you think?

Reply:
Agree, `routerLink` is more suitable there.

Reply:
but it's not going to work... (you can check) We can use routerLink only in a component template for correct execution. Angular can't handle directives of injected html. Imagine we get the html from the backend - it's almost the same. ngx-translate seems like doesn't do any convertion. As I said we have the directive which can work with links with href and do not reload the page. But I'm not sure we should overuse it. As I said I'm ok with using the link with href in json file because you already did the refactoring. But I'd prefer to have Marco's confirmation @MocanaAtBroad

Reply:
I see your point - using `RouteTransformerDirective`.
We can proceed with the directive and hard-coded href in en.json for now 
and think about how it can be improve later.

Reply:
yea, let's keep it! it would be easier for the translation team too

Reply:
Deal. 
Fixed at 86cdb7936b3d76ec0c49e4e864ce27616b65e112